modules/virt-configuring-cluster-real-time.adoc

```diff
@@ -0,0 +1,82 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/advanced_vm_management/virt-configuring-cluster-realtime-workloads.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="virt-configuring-cluster-real-time_{context}"]
+= Configuring a cluster for real-time workloads
+
+You can configure an {product-title} cluster to run real-time workloads.
+
+.Prerequisites
+* You have access to the cluster as a user with `cluster-admin` permissions.
+* You have installed the OpenShift CLI (`oc`).
+* You have installed the Node Tuning Operator.
+
+.Procedure
+
+. Label a subset of the compute nodes with a custom role; for example, `worker-realtime`:
++
+[source,terminal]
+----
+$ oc label node <node_name> node-role.kubernetes.io/worker-realtime=""
```

Comment:
This will not work in compact/SNO clusters (I don't remember why, and how to find the ticket that details it). For those, the configuration will rely on the already-existing `master` label and MCP.